dyno/microcontroller/constants.h

```diff
@@ -8,7 +8,8 @@
 const int POT_PIN = A0;
 const int NUM_TEMP_SENSE = 5;
```

Comment:
5 temperature sensors??

Reply:
It's a configurable number, but right now it's set to 5 haha. Figured we might want to attach some to the motor controller and motor itself, though that might be more DAQ territory. I think the PCB right now has support for 5 temperature sensors though, so the option is there.